src/Server.php

```diff
@@ -111,6 +111,17 @@ public function __construct(FilesystemOperator $source, FilesystemOperator $cach
         $this->tempDir = sys_get_temp_dir();
     }
 
+    private function trimPrefix(string $prefix): string
+    {
+        if ('//' == substr($prefix, -2)) {
```

Comment:
Why not check for `'://' === substr($prefix, -3)` instead of having a separate check for `:`?

Reply:
Can you respond to above query please.